README.md

```diff
@@ -71,12 +71,13 @@ dotnet publish -c Release
 ##### You have supported hardware and drivers and [NVIDIA-Docker].
 
 ```
-git clone https://github.com/NVIDIA/model-analyzer
-docker build -f Dockerfile -t model-analyzer .
+git clone https://github.com/triton-inference-server/model_analyzer.git
+cd src
```

Comment:
Could you remove this line, since the build should happen in the root folder? Actually, this should probably be `cd model_analyzer` (same for line 61: `cd model_analyzer/src`). I’ll approve after.

Reply:
Updated, thanks @dyastremsky